racket/src/cs/core.sls

```diff
@@ -566,20 +567,28 @@
           unsafe-f80vector-set!
           unsafe-f80vector-ref
 
-	  ;; future scheduler functions
-
-	  future
-	  touch
-	  future?
-	  futures-enabled?
-	  current-future
-	  would-be-future
-	  processor-count
+	  ;; pthread things
```

Comment:
I think we should consistently add "; not exported to Racket" to help others navigate the code, but maybe it makes sense to separate this group and make it a comment for the whole group. (The comment just one the last one here seems misleading.)